stable/apm-server/values.yaml

```diff
@@ -41,9 +41,9 @@ service:
 
 
 config:
-  apm-server: {}
+  apm-server:
     ### Defines the host and port the server is listening on
-    # host: "localhost:8200"
+    host: "0.0.0.0:8200"
```

Comment:
I think since this is a crucial part of how the service works, we might move it to the template. What do you think?

Reply:
I think allowing the flexibility to change this value can help. Ideally people should have the ability to change the port here as well. So I would keep this as is in the values file.